src/quo2/components/colors/color_picker/view.cljs

```diff
@@ -5,25 +5,29 @@
             [reagent.core :as reagent]
             [quo2.components.colors.color-picker.style :as style]))
 
-(def color-list [:blue :yellow :turquoise :copper :sky :camel :orange :army :pink :purple :magenta])
+;; TODO: using this to keep alignment of colors correct while b & w is being developed.
+;; https://github.com/status-im/status-mobile/issues/15442
+(def empty-color :no-color)
```

Comment:
if I used `nil` instead there is an issue with `key` prop and then there is warnings in the console.

Reply:
Quick Question: Is it fine to remove the `empty-color` and add the same comment on top of `color-list`?

Because we are making so many conditions/adjustments in our code for this one color.

Reply:
sure can do 👌 . As long as it's highlighted that is added with intention I think it's fine.